ads/_config.js

```diff
@@ -169,6 +170,12 @@ export const adPreconnect = {
     'https://output.nend.net',
     'https://img1.nend.net',
   ],
+  adcrops: [
+    'https://js.adcrops.net',
```

Comment:
You removed the names that are needed. Only `https://js.adcrops.net` should be removed here, because it is already being prefetched.